techniques/failures/F39.html

```diff
@@ -123,24 +108,21 @@ <h3>Example 1: Decorative images that have no <code class="language-html">alt</c
       </section>
    </section>
 
-<section id="related"><h2>Related Techniques</h2><ul>
+<section id="related"><h2>関連テクニック</h2><ul>
       <li><a href="../html/H37">H37: Using alt attributes on img elements</a></li>
       <li><a href="../css/C9">C9: Using CSS to include decorative images</a></li>
       <li><a href="../failures/F38">F38: Failure of Success Criterion 1.1.1 due to not marking up decorative images in HTML in a way that allows assistive technology to ignore them</a></li>
    </ul></section>
-<section id="tests"><h2>Tests</h2>
-      <section class="procedure" id="procedure"><h3>Procedure</h3>
+<section id="tests"><h2>検証</h2>
+      <section class="procedure" id="procedure"><h3>手順</h3>
          <ol>
-            <li> Identify any img elements that are used for decoration, spacing
-              or other purpose that is not part of the meaningful content in the
-              page </li>
-            <li> Check that the alt attribute for these elements is null. </li>
+            <li>装飾、スペーシング、又はページ内の意味のあるコンテンツの一部ではない他の目的に使用される img 要素を特定する</li>
+            <li>これらの要素に対する alt の値が空であることを確認する。</li>
```

Comment:
check that ... の訳を、他のテクニックと合わせました。
```suggestion
            <li>これらの要素に対する alt の値が空であることをチェックする。</li>
```